frontend/src/pages/groups.jsx

```
@@ -46,87 +28,114 @@ const Groups = ({ user, onLogout, darkMode, toggleDarkMode }) => {
         initial={{ opacity: 0, y: -20 }}
         animate={{ opacity: 1, y: 0 }}
         transition={{ duration: 0.3 }}
         className="bg-white dark:bg-slate-800 shadow-md z-10"
       >
         <div className="flex justify-between items-center  pr-4 pl-4 pt-2 pb-2">
           {isMobile ? (
-            <button onClick={toggleMobileMenu} className="p-2 rounded-full mr-2">
+            <button
+              onClick={toggleMobileMenu}
+              className="p-2 rounded-full mr-2"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)